packages/usa-input-mask/src/index.js

```diff
@@ -1,124 +1,1095 @@
+    if (FORMAT_CHARACTERS.indexOf(MASK[i]) === -1) {
+      break;
+    } else {
+      strInitVal += MASK[i];
+    }
+  }
+
+  return strInitVal;
+};
+
+/**
+ * Sets the initial mask value for a given input element.
+ *
+ * @param {HTMLElement} inputEl - The input element to set the initial mask value for.
+ */
+const setInitMask = (inputEl, isFocus = true) => {
+  const el = inputEl;
+  let position = 0;
+  let strInitVal = "";
+  const MASK = getMaskInfo(el.id, "MASK", []);
+  for (let i = 0; i < MASK.length; i += 1) {
+    if (FORMAT_CHARACTERS.indexOf(MASK[i]) === -1) {
+      position = i;
+      break;
+    } else {
+      strInitVal += MASK[i];
+    }
+  }
+
+  el.value = strInitVal;
+  setCursorPosition(inputEl, position, isFocus);
+};
+
+/**
+ * Check and insert masked characters
+ *
+ * @param {HTMLInputElement} inputEl - Input with `.usa-masked__field` class
+ * @param {number} cursorPos - Cursor position
+ */
+const changeValue = (inputEl) => {
+  const el = inputEl;
+  const strValue = getMatchValue(el.value);
+
+  let newValue = "";
+  let index = 0;
+
+  const MASK = getMaskInfo(el.id, "MASK", []);
+
+  for (let i = 0; i < MASK.length; i += 1) {
+    if (index >= strValue.length) {
+      break;
+    }
+
+    if (FORMAT_CHARACTERS.indexOf(MASK[i]) > -1) {
+      newValue += MASK[i];
+    } else if (strValue.length > index) {
+      newValue += strValue.charAt(index);
+      index += 1;
+    }
+  }
+
+  el.value = newValue;
+
+  const strNumber = getMatchValue(newValue);
+  if (strNumber === "") {
+    setInitMask(inputEl);
+  }
+};
+
+/**
+ * Modifies the value and cursor position of the given input element according to certain rules.
+ * If the input element has a value, the mask is inserted until a number is reached.
+ * If the input element has no value, it is given focus.
+ *
+ * @param {HTMLElement} inputEl - The input element to be modified.
+ * @param {number} cursorPos - The position of the cursor in the input element.
+ */
+const checkAndInsertMaskCharacters = (inputEl, cursorPos) => {
+  const el = inputEl;
+
+  changeValue(inputEl, cursorPos);
+
+  setCursorPosition(inputEl, cursorPos);
+
+  if (el.value.length > 0) {
+    insertMaskUntilNumber(inputEl, cursorPos);
+  } else {
+    el.focus();
+  }
+};
+
+/**
+ * Creates and appends two new elements to the given element: a visible status message element and a screen-reader-only status message element.
+ * Both elements have the same default text content and are given specific classes and attributes.
+ *
+ * @param {HTMLElement} maskedEl - The element to which the new status message elements will be appended.
+ */
+const createStatusMessages = (maskedEl) => {
+  const statusMessageEl = document.createElement("div");
+  const srStatusMessageEl = document.createElement("div");
+
+  statusMessageEl.classList.add(`${STATUS_MESSAGE_CLASS}`);
+  srStatusMessageEl.classList.add(
+    `${STATUS_MESSAGE_SR_ONLY_CLASS}`,
+    "usa-sr-only"
+  );
```

Comment:
We should also have a default message. Maybe the expected state or another hint.

Reply:
> Both classes seem to be identical since `${STATUS_MESSAGE_SR_ONLY_CLASS}` is defined as `usa-sr-only` on line 14.

Removed redundant class in [9d8007b](https://github.com/uswds/uswds/pull/5227/commits/9d8007b48dc92b18a764dc4779ce87ee1ef4d685)